lstchain/scripts/onsite/onsite_create_calibration_file.py

```diff
@@ -79,23 +82,35 @@ def main():
             print(f">>> Error: The pedestal file {pedestal_file} do not exist.\n Exit")
```

Comment:
I think we should be able to create calibration files using drs4 pedestal files from another different day. Maybe using a glob somewhere here. This was somewhat inconvenient for us when using this script some time ago.